app/presenters/activity_presenter.rb

```diff
@@ -67,8 +67,18 @@ def display_title
     title
   end
 
+  def parent_title
+    return if parent.blank?
+    parent.title
+  end
+
   def level
     return if super.blank?
-    I18n.t("page_content.activity.level.#{super}")
+    case super
+    when "fund" then "Fund"
+    when "programme" then "Programme"
+    when "project" then "Project"
+    when "third_party_project" then "Third-party project"
```

Comment:
The hyphen in 'third-party' here is why this method doesn't simply use `titleize`.